.github/workflows/release.yaml

```diff
@@ -55,3 +55,14 @@ jobs:
 
       - name: Create deb repository
         run: ci/deploy-deb.sh
+
+      - id: get_version
+        uses: battila7/get-version-action@v2
+
+      - name: Trigger ${{ github.repository_owner }}/trivy-action
```

Comment:
I'd use the official repository as much as possible. Can we use `actions/github-script` instead?
https://github.com/orgs/community/discussions/26323#discussioncomment-3251451

Reply:
done